ui/app/adapters/application.js

```diff
@@ -10,7 +10,9 @@ export default class ApplicationAdapter extends RESTAdapter {
   flashMessages;
 
   namespace = 'v1';
-  host = config.conduitAPIURL;
+  host = config.conduitAPIURL
+    ? config.conduitAPIURL
+    : `${window.location.protocol}//${window.location.host}`;
```

Comment:
Do we want to use the `window.location.host` here as well or is `window.location.protocol` already returning the expected value?

Reply:
I think it may just be the way the code format is rendered here that makes `//${window.location.host}` look as if it was a comment instead of part of the string -- nevermind then!

Reply:
Ah yeah!! looks like theres a highlighting issue here with github, I can confirm this works via @anaptfox & @devarismeroxa . I built them a Docker image using this change, and they were successfully able to deploy on DO and built pipelines start to finish using the UI remotely.

Reply:
That sounds perfect then, thank you for the outline!